docs/design.md

```diff
@@ -65,15 +66,16 @@ Datumaro is:
   Use case: explanation of network "quality", "stability", "certainty"
 - adversarial attacks on networks
 - dataset minification / reduction
-  Use case: removal of redundant information to reach the same network quality with lesser training time
+  Use case: removal of redundant information to reach the same network quality
+  with lesser training time
 - dataset expansion and filtration of additions
   Use case: add only important data
 - guidance for key frame selection for tracking ([paper](https://arxiv.org/abs/1903.11779))
   Use case: more effective annotation, better predictions
 
 ## RC 1 vision
 
-*CVAT integration*
+_CVAT integration_
```

Comment:
IMO, this should be a 3rd-level heading instead.

Reply:
Done